actions/terraform/init/action.yaml

```diff
@@ -90,13 +90,12 @@ runs:
       shell: bash
       if: always()
       working-directory: ${{ inputs.working_directory }}
-      env:
-        ARM_CLIENT_ID: ${{ inputs.arm_client_id }}
-        ARM_TENANT_ID: ${{ inputs.arm_tenant_id }}
-        ARM_SUBSCRIPTION_ID: ${{ inputs.tf_arm_subscription_id }}
-        ARM_USE_OIDC: "true"
       run: |
         terraform init -input=false \
+          -backend-config="use_oidc=true" \
+          -backend-config="use_azuread_auth=true" \
+          -backend-config="tenant_id=${{ inputs.arm_tenant_id }}" \
+          -backend-config="client_id=${{ inputs.arm_client_id }}" \
           -backend-config="subscription_id=${{ inputs.tf_arm_subscription_id }}" \
```

Comment:
The subscription was being set here and as an env var.
Removed the env block and doing all in the same block now.

Reply:
Most of these are not secrets since they are already hardcoded in multiple files; I've noticed the client_id we are keeping secret so closed the PR for now while I think of a better way to do this.